docs/architecture/porting-existing-aspnet-apps/choose-net-core-version.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Choosing the right .NET Core version
+description: How should you determine which version of .NET Core to target?
+author: ardalis
+ms.date: 11/13/2020
+---
+
+# Choosing the right .NET Core version
```

Comment:
As a rule of thumb, avoid gerunds in headings

```suggestion
# Choose the right .NET Core version
```

Reply:
Completely the opposite of Pluralsight guidance for clip/module naming... #sigh